routes/registration.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express');
+const router = express.Router();
+
+const mail = require('./mailService');
+const models = require('../database/models');
+let randomStr=require('./randomstring');
+
+const localUser = models.user_account;
+
+function validateEmail(email) {
```

Comment:
don't we already have a `validateEmail` function elsewhere? why not making it a reusable module instead?